tests/support.rs

```diff
@@ -0,0 +1,39 @@
+// Using a shared mod like this is probably not the best idea, since we have to
+// disable the `dead_code` lint, as we don't need all of the helpers from here
+// all over the place.
+#![allow(dead_code)]
```

Comment:
I think we already have some `src/test_util`, would that be a good place to move it? (Realizing after the fact this was copied over from the deleted `mod.rs` file)

Reply:
This warning might not even be relevant anymore, to be honest. I just kind of copypasta'd it blindly.

Reply:
Dropped the comment and the `#![allow(...)]` in 2d9bdb6, since tests seemed without it.